### PR TITLE
Disables plague rat, because plague rat bad

### PR DESCRIPTION
**Pull request by adamsong**
# Document the changes in your pull request

Someone can fix it if they want, but currently having 4 rats running around on lowpop with a 100% chance to infect with a plague that medbay can't cure while hiding under tables and shit so they can't be killed is bad.

# Changelog

:cl:  
rscdel: Plague rats can no longer spawn
/:cl:

### CODE REVIEW

**Comment by Anvilman6** [comment]
Conflict possible with #14302, as it touches on plague rat code as well

**Comment by adamsong** [comment]
Won't conflict, different part touched

File: code/controllers/subsystem/minor_mapping.dm [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #define REGAL_RAT_CHANCE 2
-#define PLAGUE_RAT_CHANCE 5
+#define PLAGUE_RAT_CHANCE 0
```
Comment (tattax) [inline code/controllers/subsystem/minor_mapping.dm:2]: > Someone can fix it if they want, but currently having 4 rats running around on lowpop with a 100% chance to infect with a plague that medbay can't cure while hiding under tables and shit so they can't be killed is bad.

cure is spaceallin, most basic virus curing chem, after that you can't be infected anymore only if you get cloned, also skill issue you can alt click tables to hit under them

the number is up to change, and they shouldn't spawn roundstart anymore


```suggestion
#define PLAGUE_RAT_CHANCE 1
```

**Comment by Tipy1802** [comment]
Thank you. I absolutely hate this antag. It exists just to be annoying and fighting it is near impossible because of how many get away tools it has. It contributes absolutely nothing positive to the game experience

**Review by Bibby0110 — APPROVED** [review]
Revert when plague rat expansion pr is done

**Pull request merged**